packages/react/src/components/CardEditor/CardEditForm/CardEditForm.jsx

```diff
@@ -175,24 +175,32 @@ export const hideCardPropertiesForEditor = (card) => {
       omit(attribute, ['aggregationMethods', 'grain'])
     );
   }
-  if (card.values?.hotspots) {
-    hotspotAttributes = card.values.hotspots.map((hotspot) => ({
-      ...hotspot,
-      content: {
-        ...hotspot.content,
-        attributes: hotspot.content.attributes?.map((attribute) =>
-          omit(attribute, ['aggregationMethods', 'grain'])
-        ),
-      },
-    }));
+  if (card.content?.columns) {
+    columns = card.content.columns.map((column) => omit(column, ['aggregationMethods', 'grain']));
   }
   return omit(
     attributes
       ? { ...card, content: { ...card.content, attributes } }
       : series
       ? { ...card, content: { ...card.content, series } }
-      : hotspotAttributes
-      ? { ...card, values: { ...card.values, hotspots: hotspotAttributes } }
+      : card.values?.hotspots
```

Comment:
Can you add a comment as to which cards pertain to which of these conditionals? A comment about why these shouldn't be modified would be nice too